lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -186,7 +186,23 @@
                                 options: fieldsData.preferred_language.options,
                                 persistChanges: true
                             })
-                        }
+                        }, 
+                        {
+                            view: new AccountSettingsFieldViews.TextFieldView({
+                            model: userAccountModel,
+                            title: gettext('Full Name'),
+                            valueAttribute: 'name',
+                            helpMessage: gettext(
+                                'The name that is used for ID verification and appears on your certificates. Other learners never see your full name. Make sure to enter your name exactly as it appears on your government-issued photo ID, including any non-Roman characters.'  // eslint-disable-line max-len
```

Comment:
Not sure where this diff comes from, looks unrelated to the current task. Please ignore this comment if I'm wrong

Reply:
resolved